sites/public/src/layouts/application.tsx

```diff
@@ -8,7 +8,12 @@ import { AlertBanner, AuthContext, MessageContext } from "@bloom-housing/shared-
 import { SiteHeader } from "@bloom-housing/doorway-ui-components/src/headers/SiteHeader"
 import CustomSiteFooter from "../components/shared/CustomSiteFooter"
 
-const Layout = (props) => {
+interface LayoutProps {
```

Comment:
Due to the new full height page format, we needed to be able to programmatically hide the footer. It is still visible at the bottom of the list view if you scroll down.

<img width="957" alt="Screenshot 2024-11-12 at 10 00 52 PM" src="https://github.com/user-attachments/assets/1df10082-f42d-4d1e-9711-b7813f628f9c">